apps/web/lib/org/[orgSlug]/instant-meeting/team/[slug]/[type]/getServerSideProps.ts

```diff
@@ -55,6 +55,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   return {
     props: {
+      eventData,
```

Comment:
By making this passed from getServerSideProps, we can use this to generate metadata without another extra query fetching in `generateMetadata`